bugbear.py

```diff
@@ -428,11 +429,16 @@ def visit(self, node):
 
         self.check_for_b018(node)
 
-    def visit_ExceptHandler(self, node):
+    def visit_ExceptHandler(self, node):  # noqa: C901 # too complex
```

Comment:
Wonder if it's worth creating an issue to break this up ...

Reply:
It is quite messy, with code for B013, B029, B030, B036 and B040 interwoven. Fairly straightforward to split out B013, B029 and B030 though - so I'll just go ahead and do that.